webapp/src/main/webapp/WEB-INF/conf/news/nodetypes-templates/news/views/view1.gtmpl

```diff
@@ -187,7 +187,7 @@
     </div>
     <!-- end -->
     <div class="notMemberUser">
-    <div id="newsDetails"></div>
+    <div id="newsDetailsApp"></div>
```

Comment:
this needs an UP IMHO otherwises the news details will not be displayed correctly from the File explorer

Reply:
Yep, we already have one, https://github.com/exoplatform/data-upgrade/blob/develop/data-upgrade-ecms/src/main/resources/conf/portal/configuration.xml#L81